packages/provider-elasticsearch/test/example-types/filters/tagsQuery.test.js

```diff
@@ -13,7 +13,7 @@ let {
   filter,
   buildResultQuery,
   result,
-} = tagsQuery
+} = tagsQuery(() => [])
```

Comment:
this invocation is wrong since tagsQuery takes an object - and with the maybeCall changes, you can just pass in nothing